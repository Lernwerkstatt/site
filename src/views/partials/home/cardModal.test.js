const card = require("./cardModal.hbs");

describe("cardModal.hbs", () => {
  it("should match the snapshot", () => {
    expect(
      card({
        title: "Aktuelles",
        text:
          "Lorem ipsum dolor sit amet consectetur adipisicing elit. Veritatis ea consequatur quibusdam sint laboriosam voluptates."
      })
    ).toMatchSnapshot();
  });
});
