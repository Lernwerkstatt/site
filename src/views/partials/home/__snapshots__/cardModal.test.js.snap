// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`cardModal.hbs should match the snapshot 1`] = `
"<div class="card mb-3 easy cursor-pointer position-relative border-0">
  <a href="#" data-toggle="modal" data-target="#"
    class="home-card card-body d-flex flex-column align-items-center justify-content-end text-decoration-none"
    style="background-image: url(); background-position: center; background-size: cover">
    <h3 class="d-inline-block p-2 text-dark bg-light">Aktuelles</h3>
    <p class="d-inline-block p-2 text-dark bg-light">Lorem ipsum dolor sit amet consectetur adipisicing elit. Veritatis ea consequatur quibusdam sint laboriosam voluptates.</p>
    <span class="home-card-badge badge badge-light position-absolute"></span>
  </a>
</div>
"
`;
